src/components/LoginOptions/LoginOptions.stories.mdx

```diff
@@ -0,0 +1,221 @@
+import { Canvas, Meta, Story, ArgsTable } from '@storybook/addon-docs';
+import { userEvent, within, waitFor } from '@storybook/testing-library';
+import { jest, expect } from '@storybook/jest';
+
+import {LiteralsProvider} from 'components/Literal';
+
+import LoginOptions from '.';
+import LoginOptionsDisplay from './LoginOptionsDisplay';
+
+<Meta
+  title="Composites/Login Options"
+/>
+
+# Login Options
+
+The `LoginOptions` component displays the options that are available for a user to authenticate when starting a form.
+
+## Presentation
+
+The `LoginOptionsDisplay` component shows the different buttons that can be clicked to start the authentication process.
+Next to each button, the corresponding logo is displayed.
+
+export const DisplayTemplate = ({loginAsYourselfOptions, loginAsGemachtigdeOptions}) => (
+  <LoginOptionsDisplay
+    loginAsYourselfOptions={loginAsYourselfOptions}
+    loginAsGemachtigdeOptions={loginAsGemachtigdeOptions}
+  />
+);
+
+<Canvas>
+  <Story
+    name="Display"
+    decorators={[(Story) => (
+      <form>
+        {Story()}
+      </form>
+    )]}
+    args={{
+      loginAsYourselfOptions: [
+        {
+          identifier: "digid",
+          label: "DigiD",
+          logo: {
+              href: "https://www.digid.nl/",
+              title: "DigiD",
+              imageSrc: "/digid.png"
+          },
+          url: "#",
+        },
+        {
+          identifier: "eHerkenning",
+          label: "eHerkenning",
+          logo: {
+              href: "https://www.eherkenning.nl/",
+              title: "eHerkenning",
+              imageSrc: "/eherkenning.png"
+          },
+          url: "#",
+        },
+        {
+          identifier: "eidas",
+          label: "eIDAS",
+          logo: {
+              href: "https://www.eherkenning.nl/",
+              title: "eIDAS",
+              imageSrc: "/eidas.png"
+          },
+          url: "#",
+        },
+        {
+          identifier: "anonymous",
+          label: "Begin form",
+        }
+      ],
+      loginAsGemachtigdeOptions: [{
+	      identifier: "digid_machtigen",
+	      label: "DigiD Machtigen",
+        logo: {
+            href: "https://www.digid.nl/",
+            title: "DigiD Machtigen",
+            imageSrc: "/digid.png"
+        },
+        url: "#",
+      }]
+    }}
+  >
+    {DisplayTemplate.bind({})}
+  </Story>
+</Canvas>
+
+### Props
+
+<ArgsTable of={LoginOptionsDisplay} />
+
+## Functional
+
+The `LoginOptions` component handles the 'smart' part of the component. It separates the login options between
+the ones that can be used to 'login as yourself' and the ones that can be used to log in 'on behalf of someone else'.
+It also formats the labels that will be displayed on the buttons and creates the right URL to which the user will be
+redirected upon clicking one of the buttons.
+
+export const LiteralDecorator = (Story) => (
+  <LiteralsProvider literals={{'beginText': {resolved: 'Begin Form'}}}>
+    {Story()}
```

Comment:
`<Story />` seems to work, but note that I also added some `parameter` config in the `Meta`:

```jsx
<Meta
  title="Composites/Login Options"
  component={LoginOptions}
  parameters={{
    docs: {
      source: {
        type: 'dynamic',
        excludeDecorators: true,
      }
    }
  }}
/>
```

`excludeDecorators` is crucial for accurate code previews